tests/unit/test_clidriver.py

```diff
@@ -679,6 +683,13 @@ def test_aws_with_cacert_env_var(self):
         self.assertEqual(call_args[0], (mock.ANY, 'us-east-1'))
         self.assertEqual(call_args[1]['verify'], '/path/cacert.pem')
 
+    def test_aws_with_read_timeout(self):
+        self.assert_params_for_cmd(
+            'lambda invoke --function-name foo out.log --cli-read-timeout 90',
+            expected_rc=0)
+        call_args = self.create_endpoint.call_args
+        self.assertEqual(call_args[1]['timeout'], (60, 90))
```

Comment:
One minor thing. Shall we just check `self.assertEqual(call_args[1]['timeout'][1], 90)`? Just in case we would change the default connect_timeout into something other than 60 in future.